index.html

```diff
@@ -54,10 +54,10 @@ <h1>I'm Zermarden <br>
         <img class="header-background" src="assets/header-background-mobile.svg" alt="background-image">
     </header>
     <main>
-        <section class="works">
+        <div class="works">
```

Comment:
- [ ] I kindly suggest that you make this `div` tag to a `section` tag because of semantics
- [ ] I kindly suggest that you apply a display of grid and not flex to the styling of your works container as it  is part of project requirements